### ISSUE TITLE
The core: What does it include?

### ISSUE THREAD

**Issue by distantnative** [kirby-kart/kart#2]
The first thing, after all initial structural questions are decided, would be the core.
What should it all include?

- Products
- Cart
- Checkout process
- Order management

**Comment by plaidscreen** [comment]
Products in my mind would just be a blueprint, can you think of any reason we'd need to make it more complicated than that? (Hope that doesn't sound confrontational, I'm genuinely asking)

**Comment by distantnative** [comment]
No, not much. Blueprint(s), example templates, snippets maybe. Still should be part of the core package, that's why I listed it, shouldn't it? Like on contrast to maybe product bundles or whatever.

**Comment by benseitz** [comment]
We would need special fields for price, product options and so on?

**Comment by plaidscreen** [comment]
@distantnative Yeah absolutely should be part of the core, in my head I was thinking you were talking about PHP-based core modules. But yes, Product blueprints, snippets, templates, should come with the standard Kart plugin.

**Comment by plaidscreen** [comment]
@benseitz I think those could still be covered by the blueprints, with special functionality being handled through whatever system we develop to handle event hooks & triggers.

**Comment by benseitz** [comment]
:+1:
